src/Application/Migrations/18_function_ticket_dependency_missing.sql

```diff
@@ -37,7 +37,7 @@ BEGIN
 	WHERE
 		depender.id = param_depender_ticket_id;
 
-	RETURN result;
+	RETURN COALESCE(result, true);
```

Comment:
Now I'm confused. So we change the default return value to true? But the comment above the functions explains: "In all other cases, including the cases where this check does not make any sense, it will return false."
Which one is wrong now?

Reply:
Also, I fail to reproduce the issue which is meant to be fixed with this.
What are the circumstances to reproduce this?

Reply:
The failing circumstances are: you have encoding tickets that have a depending profile as their profile. If they are e.g. in state "ready to encode" and you go into the detail view of the webapp, you do not get the green "indicator" (left of edit button) but you get an amber indicator, stating that the dependee ticket is missing while this is not true at all.

I will try to align the comments with the new behaviour.